### PR TITLE
Updated FAQs

### PR DESCRIPTION
**Pull request by jerae-duffin**
Updated Subscription and Billing FAQs.

### CODE REVIEW

File: docker-hub/billing/faq.md [hunk 1]
```diff
@@ -1,211 +1,83 @@
...
+### Do collaborator limits differ between the Free and Pro plans?
 
-### Are image layers counted?
+The Free plan includes unlimited collaborators for public repositories and 0 collaborators for private repositories. Pro includes unlimited collaborators for public repositories and 1 unique collaborator for private repositories. Note that a collaborator on a private repository within a Pro plan can only be used with a service account. For details on the differences between plans, see (https://www.docker.com/pricing).
 
-No. Because we are limiting on manifest requests, the number of layers (blob requests) related to a pull is unlimited at this time. This is a change from our previous policy based on community feedback in order to be more user-friendly, so users do not need to count layers on each image they may be using.
+### Where can I renew my Docker Enterprise subscription??
 
-### Are anonymous (unauthenticated) pulls rate-limited based on IP address?
+The former Docker Enterprise product is now Mirantis Enterprise Platform. You can contact Mirantis for more information by visiting their [website](https://www.mirantis.com/%20target=).
```
Comment (jhratliff) [inline docker-hub/billing/faq.md:50]: Is this URL valid? Seems like the "%20target=" should be something like [website](https://www.mirantis.com){:target="_blank"}

File: docker-hub/billing/faq.md [hunk 2]
```diff
@@ -214,338 +86,138 @@ For more information, view our blogs on [inactive image retention](https://www.d
...
-      }
-    },
-    {
-      "@type": "Question",
-      "name": "How does Team pricing work?",
+      "name": "What subscriptions levels are available for Docker?",
       "acceptedAnswer": {
         "@type": "Answer",
-        "text": "Team starts at $25 per month for the first 5 users and $7 per month for each user thereafter with an Annual subscription."
+        "text": "Docker Subscriptions come in three levels:\nFree plans for both individuals and development teams that include unlimited public repos.\nDocker Pro Subscriptions for individuals with unlimited private repos, unlimited public repos, no rate limiting on pull requests and premium support starting at $5/month with an annual subscription.\nDocker Team Subscriptions for development teams with unlimited private repos and unlimited public repos starting at $25 per month for the first 5 users and $7 per month for each user thereafter with an Annual subscription. The Team plan offers advanced collaboration and management tools, including organization and team management with role-based access controls.\n\n"
```
Comment (jhratliff) [inline docker-hub/billing/faq.md:92]: Probably don't need the newlines at the end of this. Not an error, just a preference.

**Review by jhratliff — CHANGES_REQUESTED** [review]
Looks good - a couple of questions (URL and newlines). Other than those... LGTM!

File: docker-hub/billing/faq.md [hunk 3]
```diff
@@ -1,211 +1,83 @@
...
 ---
 
-## Pricing FAQs
+## Subscription Tiers
 
-As of May 14, 2020, Docker announced [new pricing changes](/docker-hub/billing/) for Docker Hub subscriptions.
+### What subscriptions levels are available for Docker?
 
-### What plans and pricing changes did Docker announce on May 14?
+[Docker Subscriptions](https://www.docker.com/pricing/resource-consumption-updates) come in three levels:
```
Comment (usha-mandya) [inline docker-hub/billing/faq.md:11]: Could you add {:target="_blank" rel="noopener" class="_"} so it opens the link in a new tab?

File: docker-hub/billing/faq.md [hunk 4]
```diff
@@ -1,211 +1,83 @@
...
+* Free plans for both individuals and development teams that include unlimited public repos.
+* Docker Pro Subscriptions for individuals with unlimited private repos, unlimited public repos, no rate limiting on pull requests and premium support starting at $5/month with an annual subscription.
```
Comment (usha-mandya) [inline docker-hub/billing/faq.md:13]:
```suggestion
* Docker Pro Subscriptions for individuals with unlimited private repos, unlimited public repos, no rate limiting on pull requests and premium support starting at $5 per month with an annual subscription.
```

File: docker-hub/billing/faq.md [hunk 5]
```diff
@@ -1,211 +1,83 @@
...
-
-### I have more questions regarding the new pricing, who can I reach out to?
-
-If you have any questions about how the new seat-based pricing impacts you that are not covered in the FAQ, please reach out and a member of the Docker team will get back to you.
-
-## Pull Rate Limiting FAQs
+### What are the rate limits for free subscriptions?
 
-### What are the Docker Terms of Service?
+Anonymous and Free subscriptions are subject to rate limits on container image pull requests. You can get more information at the [Rate Limiting Information](https://docs.docker.com/docker-hub/download-rate-limit/#how-do-i-authenticate-pull-requests%20target=) page.
```
Comment (usha-mandya) [inline docker-hub/billing/faq.md:26]: This should be a relative link to the Rate limits topic

File: docker-hub/billing/faq.md [hunk 6]
```diff
@@ -1,211 +1,83 @@
...
+### Does Docker offer academic pricing?
 
-### Are pulls from logged-in accounts rate-limited based on IP address?
+Contact the [Docker Sales Team](https://www.docker.com/company/contact).
 
-No, limits on pull requests that are [authenticated with an account](/docker-hub/download-rate-limit/#how-do-i-authenticate-pull-requests) are based on that account and not on the IP. Free accounts are limited to 200 pulls per 6 hour period. Paid accounts are subject to fair usage as outlined in [Docker Pricing](https://www.docker.com/pricing){: target="blank" rel="noopener" class=""}.
+### Can Docker Hub be used for free for open source projects?
 
-### Will I be rate-limited if I’m logged in but someone else on my IP hits the anonymous limit?
+Docker is a proud open source software company, and provides qualifying OSS software projects with access to Docker Hub with unlimited public repositories unlimited consumption of OSS project container images and unlimited collaborators at no cost. For questions about open source projects and pricing please visit the [Open Source Community Application](https://www.docker.com/community/open-source/application).
```
Comment (usha-mandya) [inline docker-hub/billing/faq.md:58]:
```suggestion
Docker is a proud open source software company, and provides qualifying OSS software projects with access to Docker Hub with unlimited public repositories, unlimited consumption of OSS project container images, and unlimited collaborators at no cost. For questions about open source projects and pricing, visit the [Open Source Community Application](https://www.docker.com/community/open-source/application).
```

File: docker-hub/billing/faq.md (hunk 2 above)
Comment (jerae-duffin) [reply docker-hub/billing/faq.md:92]: Should I remove the \n from the lines?

File: docker-hub/billing/faq.md (hunk 1 above)
Comment (jerae-duffin) [reply docker-hub/billing/faq.md]: Updated with new URL

**Comment by jerae-duffin** [comment]
@usha-mandya  I was able to push the changes, however there are conflicts and cannot merge pull request.

**Comment by jerae-duffin** [comment]
@usha-mandya  Resolved all Errors and updated links. Ready for Merge PR.

**Comment by usha-mandya** [comment]
@jerae-duffin Looks like the updates added as part of annual renewals are missing from this PR. See https://github.com/docker/docker.github.io/pull/12931.

**Comment by jerae-duffin** [comment]
@usha-mandya  Updated PR with all FAQs and links aligned with WWW.

File: docker-hub/billing/faq.md [hunk 7]
```diff
@@ -1,223 +1,235 @@
...
+### What happens if I pay for an annual subscription and add more users later?
 
-For information on how to add paid seats to a monthly plan for your organization, see [Add seats to a monthly plan](/docker-hub/billing/add-seats/).
+When you add seats to your plan in the middle of your billing cycle, a prorated amount will be charged for the additional seats.
 
-For information on how to remove paid seats from a monthly plan for your organization, see [Remove seats from a monthly plan](/docker-hub/billing/remove-seats/).
+### How do I upgrade to a Pro plan from a legacy individual plan?
 
-### I want to run an automated agent that makes container requests on behalf of my organization. Which license do I need?
+Upgrading your legacy plan to a Pro plan offers you unlimited public and private repos and 1 collaborator or service account. For information on how to upgrade to a Pro plan from a legacy plan, see the [How To Upgrade](https://docs.docker.com/docker-hub/billing/upgrade/y) page.
```
Comment (usha-mandya) [inline docker-hub/billing/faq.md:94]: Could you add relative links for internal docs URLs on this page?

File: docker-hub/billing/faq.md [hunk 8]
```diff
@@ -1,223 +1,237 @@
 ---
-description: Pricing & Pull Rate Limiting FAQs
-keywords: Docker, Docker Hub, pricing FAQs, pull rate limiting FAQs, subscription, platform
-title: Pricing & Pull Rate Limiting FAQs
+description: Subscription & Billing FAQs
+keywords: Docker, Docker Hub, billing FAQs, pull rate limiting FAQs, subscription, platform
+title: Subscription & Billing FAQs
+toc_min: 1
+toc_min: 2
```
Comment (usha-mandya) [inline docker-hub/billing/faq.md:6]: This should be toc_max

File: docker-hub/billing/faq.md [hunk 9]
```diff
@@ -1,223 +1,237 @@
...
 ---
 
-## Pricing FAQs
+## Subscription Tiers
```
Comment (usha-mandya) [inline docker-hub/billing/faq.md:9]: Subscription tiers

File: docker-hub/billing/faq.md [hunk 10]
```diff
@@ -1,223 +1,237 @@
...
+Contact the [Docker Sales Team](https://www.docker.com/company/contact){:target="blank" rel="noopener" class=""}.
 
-### How can I create a new Docker Hub account?
+### Can Docker Hub be used for free for open source projects?
 
-You can create a new account at [Docker Pricing](https://www.docker.com/pricing){: target="blank" rel="noopener" class=""} where you can choose a plan for Individuals or a plan for Teams.
+Docker is a proud open source software company, and provides qualifying OSS software projects with access to Docker Hub with unlimited public repositories, unlimited consumption of OSS project container images, and unlimited collaborators at no cost. For questions about open source projects and pricing, visit the [Open Source Community Application](https://www.docker.com/community/open-source/application){:target="blank" rel="noopener" class=""}.
 
-### How do I upgrade to a Pro plan from a legacy individual plan?
+## Billing Questions
```
Comment (usha-mandya) [inline docker-hub/billing/faq.md:62]: Billing questions

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**